### PR TITLE
User testing tweaks

### PR DESCRIPTION
**Pull request by hammerfunk**
Placeholder and highlight for hardcoded dates
Copy tweaks
Tweak password generator to exclude the email
Sign up COPY

### CODE REVIEW

File: app/assets/javascripts/frontend/password-strength-indicator.js [hunk 1]
```diff
@@ -8,10 +8,11 @@
   var PasswordStrengthIndicator = function(options) {
     var instance = this;
 
-    $.each([options["password_field"], options["password_confirmation_field"]], function(i, password_field) {
+    $.each([options["password_field"], options["password_confirmation_field"], options["email_field"]], function(i, password_field) {
```
Comment (matthewford) [inline app/assets/javascripts/frontend/password-strength-indicator.js]: Line is too long.<br>['password_field'] is better written in dot notation.<br>['password_confirmation_field'] is better written in dot notation.<br>['email_field'] is better written in dot notation.<br>Identifier 'password_field' is not in camel case.

File: app/assets/javascripts/frontend/password-strength-indicator.js [hunk 2]
```diff
@@ -8,10 +8,11 @@
...
       var update = function() {
         var password = $(options["password_field"]).val(),
             passwordConfirmation = $(options["password_confirmation_field"]).val();
+        options["weak_words"] = $(options["email_field"]).val().split(/\W+/);
```
Comment (matthewford) [inline app/assets/javascripts/frontend/password-strength-indicator.js]: ['weak_words'] is better written in dot notation.<br>['email_field'] is better written in dot notation.

File: app/assets/javascripts/frontend/password-strength-indicator.js [hunk 3]
```diff
@@ -87,6 +88,7 @@ $(function() {
       password_strength_guidance: $('#password-guidance'),
       password_confirmation_field: $passwordConfirmationField,
       password_confirmation_guidance: $('#password-confirmation-guidance'),
+      email_field: $emailField,
```
Comment (matthewford) [inline app/assets/javascripts/frontend/password-strength-indicator.js]: Identifier 'email_field' is not in camel case.

File: app/assets/stylesheets/admin/base.scss [hunk 4]
```diff
@@ -208,6 +208,41 @@ table .ellipsis {
...
+    left: 20px;
+  }
+
+  &.selectable-group {
+    .selectable {
+      padding-left: 40px;
+    }
+
+    .checkbox {
+      left: 15px;
```
Comment (matthewford) [inline app/assets/stylesheets/admin/base.scss]: Properties should be ordered top, left, margin-top

File: app/assets/stylesheets/frontend/forms.scss [hunk 5]
```diff
@@ -19,43 +19,54 @@ $black-true: #000;
...
+    left: $gutter-two-thirds;
+  }
+
+  &.selectable-group {
+    .selectable {
+      padding-left: 40px;
+    }
+
+    .checkbox {
+      left: 15px;
```
Comment (matthewford) [inline app/assets/stylesheets/frontend/forms.scss]: Properties should be ordered top, left

File: app/assets/stylesheets/frontend/forms.scss [hunk 6]
```diff
@@ -19,43 +19,54 @@ $black-true: #000;
...
 }
 
-.page-devise .boolean .error {
-  margin-left: 0;
+h2 > .boolean.control-label {
+  padding-left: 0;
+}
+
+.input:after {
+  content: "\0020";
```
Comment (matthewford) [inline app/assets/stylesheets/frontend/forms.scss]: Properties should be ordered display, height, clear, content, overflow

File: app/assets/stylesheets/frontend/forms.scss [hunk 7]
```diff
@@ -891,6 +902,55 @@ input[type="file"] {
...
+  }
+}
+
+.field-with-errors .error,
+.smart-answer article .error,
+.question-has-errors .errors-container li,
+.supporters-list .errors-container li,
+.audit_certificate_upload_form .errors-container li {
+  @include core-19;
+  clear: both;
```
Comment (matthewford) [inline app/assets/stylesheets/frontend/forms.scss]: Properties should be ordered display, margin, margin-bottom, padding, clear, border, background, color

File: app/assets/stylesheets/frontend/forms.scss [hunk 8]
```diff
@@ -891,6 +902,55 @@ input[type="file"] {
...
+  margin-bottom: 0.2em;
+  padding: 0;
+  list-style: none;
+
+  &.char-spacing {
+    padding-right: 160px;
+  }
+}
+
+.field-with-errors .error,
```
Comment (matthewford) [inline app/assets/stylesheets/frontend/forms.scss]: Selector `audit_certificate_upload_form` should be written in lowercase with hyphens

File: app/forms/qae_2014_forms/enterprise_promotion/enterprise_promotion_step6.rb [hunk 9]
```diff
@@ -75,7 +75,7 @@ def promotion_step6
               If you've answered all the questions you can now submit your nomination.
             </p>
             <p>
-              You can still edit your submitted nomination at any time before 23:59 on the last working day of September.
+              You can still edit your submitted nomination at any time before <span class='todo-placeholder'>23:59 on the last working day of September</span>.
```
Comment (matthewford) [inline app/forms/qae_2014_forms/enterprise_promotion/enterprise_promotion_step6.rb]: Line is too long. [159/100]

File: app/forms/qae_2014_forms/innovation/innovation_step1.rb [hunk 10]
```diff
@@ -55,14 +55,16 @@ def innovation_step1
           style "small"
         end
 
+        # TODO: Hardcoded date
         date :started_trading, "Date started trading" do
           required
           ref "A 5"
-          context "<p>Organisations that began trading after 01/10/2012 aren't eligible for this award.</p>"
+          context "<p>Organisations that began trading after <span class='todo-placeholder'>01/10/2012</span> aren't eligible for this award.</p>"
```
Comment (matthewford) [inline app/forms/qae_2014_forms/innovation/innovation_step1.rb]: Line is too long. [146/100]

File: app/forms/qae_2014_forms/innovation/innovation_step1.rb [hunk 11]
```diff
@@ -55,14 +55,16 @@ def innovation_step1
...
           date_max "01/10/2012"
         end
 
-        options :queen_award_holder, "Are you a current Queen's Award holder (2010-2014)?" do
+        # TODO: Hardcoded date
+        options :queen_award_holder, "Are you a current Queen's Award holder <span class='todo-placeholder'>(2010-2014)</span>?" do
```
Comment (matthewford) [inline app/forms/qae_2014_forms/innovation/innovation_step1.rb]: Line is too long. [131/100]

File: app/forms/qae_2014_forms/innovation/innovation_step6.rb [hunk 12]
```diff
@@ -74,7 +74,7 @@ def innovation_step6
         submit "Submit application" do
           notice %(
             <p>
-              If you have answered all the questions, you can submit your application now. You will be able to edit it any time before 23:59 on the last working day of September.
+              If you have answered all the questions, you can submit your application now. You will be able to edit it any time before <span class='todo-placeholder'>23:59 on the last working day of September</span>.
```
Comment (matthewford) [inline app/forms/qae_2014_forms/innovation/innovation_step6.rb]: Line is too long. [216/100]

File: app/forms/qae_2014_forms/international_trade/international_trade_step1.rb [hunk 13]
```diff
@@ -53,14 +53,16 @@ def trade_step1
           style "small"
         end
 
+        # TODO: Hardcoded date
         date :started_trading, "Date started trading" do
           required
           ref "A 5"
-          context "<p>Organisations that began trading after 01/10/2012 aren't eligible for this award.</p>"
+          context "<p>Organisations that began trading after <span class='todo-placeholder'>01/10/2012</span> aren't eligible for this award.</p>"
```
Comment (matthewford) [inline app/forms/qae_2014_forms/international_trade/international_trade_step1.rb]: Line is too long. [146/100]

File: app/forms/qae_2014_forms/international_trade/international_trade_step1.rb [hunk 14]
```diff
@@ -53,14 +53,16 @@ def trade_step1
...
           date_max "01/10/2012"
         end
 
-        options :queen_award_holder, "Are you a current Queen's Award holder (2010-2014)?" do
+        # TODO: Hardcoded date
+        options :queen_award_holder, "Are you a current Queen's Award holder <span class='todo-placeholder'>(2010-2014)</span>?" do
```
Comment (matthewford) [inline app/forms/qae_2014_forms/international_trade/international_trade_step1.rb]: Line is too long. [131/100]

File: app/forms/qae_2014_forms/international_trade/international_trade_step6.rb [hunk 15]
```diff
@@ -73,7 +73,7 @@ def trade_step6
         submit "Submit application" do
           notice %(
             <p>
-              If you've answered all the questions, you can submit your application now. You will be able to edit it any time before 23:59 on the last working day of September.
+              If you've answered all the questions, you can submit your application now. You will be able to edit it any time before <span class='todo-placeholder'>23:59 on the last working day of September</span>.
```
Comment (matthewford) [inline app/forms/qae_2014_forms/international_trade/international_trade_step6.rb]: Line is too long. [214/100]

File: app/forms/qae_2014_forms/sustainable_development/sustainable_development_step1.rb [hunk 16]
```diff
@@ -58,12 +58,13 @@ def development_step1
...
         end
 
+        # TODO: Hardcoded date
         date :started_trading, "Date started trading" do
           required
           ref "A 5"
           context %(
             <p>
-              Organisations that began trading after 01/10/2012 aren't eligible for this award.
+              Organisations that began trading after <span class='todo-placeholder'>01/10/2012</span> aren't eligible for this award.
```
Comment (matthewford) [inline app/forms/qae_2014_forms/sustainable_development/sustainable_development_step1.rb]: Line is too long. [133/100]

File: app/forms/qae_2014_forms/sustainable_development/sustainable_development_step1.rb [hunk 17]
```diff
@@ -72,7 +73,8 @@ def development_step1
         header :business_awards_header, "Business awards" do
         end
 
-        options :queen_award_holder, "Are you a current Queen's Award holder (2010-2014)?" do
+        # TODO: Hardcoded date
+        options :queen_award_holder, "Are you a current Queen's Award holder <span class='todo-placeholder'>(2010-2014)</span>?" do
```
Comment (matthewford) [inline app/forms/qae_2014_forms/sustainable_development/sustainable_development_step1.rb]: Line is too long. [131/100]

File: app/forms/qae_2014_forms/sustainable_development/sustainable_development_step2.rb [hunk 18]
```diff
@@ -53,7 +53,7 @@ def development_step2
           classes "sub-question"
           context %(
             <p>
-              Environmental dimensions' means respecting the limits of the planet’s environment, resources and biodiversity
+              'Environmental dimensions' means respecting the limits of the planet’s environment, resources and biodiversity
```
Comment (matthewford) [inline app/forms/qae_2014_forms/sustainable_development/sustainable_development_step2.rb]: Line is too long. [124/100]

File: app/forms/qae_2014_forms/sustainable_development/sustainable_development_step6.rb [hunk 19]
```diff
@@ -73,7 +73,7 @@ def development_step6
         submit "Submit application" do
           notice %(
             <p>
-              If you have answered all the questions, you can submit your application now. You will be able to edit it any time before 23:59 on the last working day of September.
+              If you have answered all the questions, you can submit your application now. You will be able to edit it any time before <span class='todo-placeholder'>23:59 on the last working day of September</span>.
```
Comment (matthewford) [inline app/forms/qae_2014_forms/sustainable_development/sustainable_development_step6.rb]: Line is too long. [216/100]

File: app/assets/stylesheets/frontend/forms.scss (hunk 8 above)
Comment (hammerfunk) [inline app/assets/stylesheets/frontend/forms.scss]: Fixed

**Pull request merged**